### ISSUE TITLE
Possible mismatch in runtime weight scaling implementation (equalized learning rate section)

### ISSUE THREAD

**Issue by akanimax** [nashory/pggan-pytorch#32]
In your code here: [`x = self.conv(x.mul(self.scale))`](https://github.com/nashory/pggan-pytorch/blob/master/custom_layers.py#L113), the input `x` is multiplied by the `scale` which is equal to `scale = sqrt(2 / fan_in)` from HE initializer. I am a bit confused about the multiplication. The paper states that `w_i_hat = w / scale` which in case of convolution, can be achieved by doing `out = conv(x / scale)`. 

My question is: why is the scale multiplied by the x, instead of dividing? Please help.
